spacy_experimental/biaffine_parser/arc_predicter.pyx

```diff
@@ -43,38 +44,48 @@ DEFAULT_ARC_PREDICTER_MODEL = Config().from_str(default_model_config)["model"]
     assigns=["token.head"],
     default_config={
         "model": DEFAULT_ARC_PREDICTER_MODEL,
-        "scorer": {"@scorers": "spacy-experimental.biaffine_parser_scorer.v1"}
+        "scorer": {"@scorers": "spacy-experimental.biaffine_parser_scorer.v1"},
+        "senter": None,
+        "max_length": 100,
     },
 )
 def make_arc_predicter(
     nlp: Language,
     name: str,
     model: Model,
     scorer: Optional[Callable],
+    senter: Optional[str],
+    max_length: int,
 ):
-    return ArcPredicter(nlp.vocab, model, name, scorer=scorer)
+    return ArcPredicter(nlp, model, name, max_length=max_length, scorer=scorer, senter=senter)
 
 
 class ArcPredicter(TrainablePipe):
```

Comment:
It's a spelling variation? https://www.collinsdictionary.com/dictionary/english/predicter

But since we already have active users of the current biaffine parser, I don't think we should change it (unless if the spelling was wrong).

Reply:
We agreed to rename it to `Predictor` many months ago, to go with `SpanPredictor`. I think we should still go through with the renaming, just not in this PR, and it should be at a new minor version.

Reply:
Oh, I wasn't aware of any naming discussion? At any rate, if there is `SpanPredictor` it makes sense to rename it for consistency. But it's annoying, since the biaffine parser actually has downstream users already (even though I recommended them to wait until all these changes are done).